lib/spatial_features/utils.rb

```diff
@@ -52,5 +52,10 @@ def id_sql(object)
         object.unscope(:select).select(:id).to_sql
       end
     end
+
+    # Convert a hash of GeoJSON data into a PostGIS geometry object
+    def geom_from_json(geometry)
```

Comment:
I wonder if we go one step further and make a util method that is just "Call this sql method and return the value" since we do that a lot in this gem.

Reply:
I extracted a `select_db_value` helper on a new commit.  It's almost nice... there are still a few straggling calls to `select_one` and `select_all` that are kind of irksome.   What do you think?